.github/workflows/daily-builds.yml

```diff
@@ -53,8 +59,20 @@ jobs:
           cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Debug -GNinja -DCMAKE_CXX_COMPILER_LAUNCHER=ccache ${{ matrix.flags }}
           cd ${{github.workspace}}/build && pwd
           du -hcs _deps/
-      - name: Build
+      - name: Build & Run
         run: |
           cd ${{github.workspace}}/build
           ninja src/all
           ccache --show-stats
+          ./dragonfly --alsologtostderr --default_lua_flags=allow-undeclared-keys &
+      - name: Clone and build BullMQ
+        run: |
+          cd ${{github.workspace}}
+          git clone https://github.com/taskforcesh/bullmq.git
+          cd bullmq
+          yarn install --ignore-engines --frozen-lockfile --non-interactive
+          yarn build
+      - name: Test BullMQ with dragonfly
+        run: |
+          cd ${{github.workspace}}/bullmq
+          yarn test
```

Comment:
we want to exclude one test that we have a problem with
please run
yarn test -i  -g "should process delayed jobs with several workers respecting delay"

Reply:
Should I add a new job "bullmq" then?